core/templates/dev/head/pages/community-dashboard-page/services/translate-text.service.ts

```diff
@@ -62,16 +67,12 @@ angular.module('oppia').factory('TranslateTextService', [
         stateNamesList = [];
         activeExpId = expId;
         activeExpVersion = null;
-        $http.get(
-          '/gettranslatabletexthandler', {
-            params: {
-              exp_id: expId,
-              language_code: languageCode
-            }
-          }).then(
-          function(response) {
-            stateWiseContents = response.data.state_names_to_content_id_mapping;
-            activeExpVersion = response.data.version;
+        TranslateTextBackendService.getTranslatableText({
+          expId, languageCode
+        }).then(
+          function(data) {
```

Comment:
The params over here should be (`stateNamesToContentIdMapping`, `expVersion`)

Reply:
You mean these variables should be there instead of data?